index.css

```diff
@@ -482,6 +482,53 @@ html * {
   .full-screen-blur {
     filter: blur(0.3rem);
   }
+
+  .modal {
```

Comment:
Great job team 🤝

- [ ] Kindly make sure your popup is responsive enough. When I try to reduce the screen size, the Image appears too small compared to the rest of the page. 
- [ ] Also, some elements of your design seem to be overflowing off of the design. You may kindly fix this
![responsiveness](https://user-images.githubusercontent.com/63917725/209414397-8b1b1e8d-d2ec-4c72-900b-4346f94f15da.PNG)

- [ ] Kindly make sure that the design of the desktop popup is well aligned so that all elements are visible and the image is not blurred.
![desktop popup](https://user-images.githubusercontent.com/63917725/209414484-9ce119a4-03f0-46c3-b119-e908513cd986.PNG)

Reply:
Thanks for the feedback. I am changing the properties to match the template